pkg/apis/apis.go

```diff
@@ -4,13 +4,13 @@ import (
 	"github.com/codeready-toolchain/api/pkg/apis"
 
 	routev1 "github.com/openshift/api/route/v1"
-	templatev1 "github.com/openshift/api/template/v1"
+	extensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
 // AddToScheme adds all Resources to the Scheme
 func AddToScheme(s *runtime.Scheme) error {
-	addToSchemes := append(apis.AddToSchemes, templatev1.Install)
```

Comment:
Don't we really need the template schema anymore?

Reply:
look at the next line that was there: 
`addToSchemes = append(apis.AddToSchemes, routev1.Install)`
the template schema wasn't used there before, there was a small bug - when appending the `routev1.Install` there was used `apis.AddToSchemes` instead of `addToSchemes` so it overrode the original `addToSchemes` that contained the template schema. 
Actually, when I tried to fix it and add the template schema then it started failing. I'm not sure why, but as it's working with this setup I decided to leave it as it is and investigate the failure later.

Reply:
hm.. ok